exporter/clickhousetracesexporter/options.go

```diff
@@ -48,7 +48,7 @@ const (
 
 const (
 	suffixEnabled         = ".enabled"
-	suffixDatasource      = ".datasource"
+	suffixDatasources     = ".datasources"
```

Comment:
I'm unclear on the best path forward here, re: naming, if my changes are adopted:

1. Continue to use `datasource:`, even though it can now be plural;
2. Switch names to be more semantic, but break backwards compat;
3. Match the other exporter(s) and use `DSN:` or even `DSNs:`?